app/models/spree/tax_rate.rb

```diff
@@ -105,15 +105,6 @@ def compute_amount(item)
         if default_zone_or_zone_match?(item.order)
           calculator.compute(item)
         else
-          # Tax refund should not be possible with the way our production server are configured
-          Alert.raise(
-            "Notice: Tax refund should not be possible, please check the default zone and " \
-            "the tax rate zone configuration"
-          ) do |payload|
-            payload.add_metadata :order_tax_zone, item.order.tax_zone
-            payload.add_metadata :tax_rate_zone, zone
-            payload.add_metadata :default_zone, Zone.default_tax
-          end
           # In this case, it's a refund.
```

Comment:
Thanks to your investigation, we now know why this is here :D
```suggestion
          # In this case, it's a refund (for instance offering a manual discount via an adjustment)
```